editor/plugins/skeleton_3d_editor_plugin.cpp

```diff
@@ -719,11 +719,7 @@ bool Skeleton3DEditor::can_drop_data_fw(const Point2 &p_point, const Variant &p_
 	}
 
 	const String path2 = target->get_metadata(0);
-	if (!path2.begins_with("bones/")) {
-		return false;
-	}
-
-	return true;
+	return !!path2.begins_with("bones/");
```

Comment:
```suggestion
	return path2.begins_with("bones/");
```

unecessary exclamation marks